search.php

```diff
@@ -0,0 +1,62 @@
+<?php
+if(isset($_GET["term"])){
+  $terms = $_GET["term"];
+}
+else{
+   echo  "you must need to enter a search term";
+}
+
+?>
+<!DOCTYPE html>1
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Google</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+
+    <!-- Main Part Start -->
+    <div class="header-container">
+        <div class="container-main">
+            <div class="header-logo">
+                <img src="https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_92x30dp.png" alt="">
+            </div>
+            <div class="header-search-container">
+                <form action="search.php" method="GET">
+                    <input class="searchbox" type="text" name="term">
+                    <button type="button" value="search" class="search_btn">search
+                    </button>
+                </form>
+            </div>
+
+            <div class="tabs-container">
+                <ul class="all_ta
+                bs">
+                    <li><a href='<?php echo "search.php?term=$terms&type=sites"; ?>'>All</a>
```

Comment:
Any better way to do it?

Reply:
There are different approaches to do it, but maybe it's fine for now if you want you can also use a short-handed format as explained here: https://www.ntchosting.com/encyclopedia/scripting-and-programming/php/php-in/#PHP_in_HTML_using_short_open_tag

Reply:
Also: does the $terms variable adds value properly there? I think you might need to use the `{$terms}` for variable interpolation. Here is the PHP doc: https://www.php.net/manual/en/language.types.string.php#language.types.string.parsing

Reply:
> Also: does the $terms variable adds value properly there? I think you might need to use the `{$terms}` for variable interpolation. Here is the PHP doc: https://www.php.net/manual/en/language.types.string.php#language.types.string.parsing

I used it for getting the value from searching and it's working properly.. Is it better to use {$terms} rather than $terms?

Reply:
> There are different approaches to do it, but maybe it's fine for now if you want you can also use a short-handed format as explained here: https://www.ntchosting.com/encyclopedia/scripting-and-programming/php/php-in/#PHP_in_HTML_using_short_open_tag

There are some shorthand formats where they are using some short tags.. but should I need to rely on it for building a compatible website?